client/src/components/atoms/SearchableDropdown.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FunctionComponent, useState } from "react";
+import { Tag, TagProps } from "../atoms/Tag";
+import ScrollWindow from "../atoms/ScrollWindow";
+import { TextField } from "../atoms/TextField";
+
+interface Props {
+  placeholderText: string,
+  searchPlaceholderText: string,
+  noResultsText: string,
+  noResultsActionText: string,
+  dropdownItemsText: Array<string>,
```

Comment:
Not a fan of this approach. Lots of duplication. Perhaps pass in a function to convert objects to dropdown items and a function that is used in filter for whatever is passed in?? Idk

At least dropdownItemsText should be optional (dropdownItemsText?)

I think the simplest approach may be to just create Tags from dropdownItemsText if a boolean displayAsTags is True.

Reply:
good call! made the change